core/lib/zksync_core/src/api_server/web3/backend_jsonrpsee/namespaces/zks.rs

```diff
@@ -174,4 +175,30 @@ impl ZksNamespaceServer for ZksNamespace {
         self.get_base_token_l1_address_impl()
             .map_err(|err| self.current_method().map_err(err))
     }
+
+    async fn send_raw_transaction_optimistic(
```

Comment:
I'm not sure about the naming here. I know that Lens refer to this API as `Optimistic`, but in reality it only refers to the fact that we return VM logs/events that result from the VM execution. There is nothing optimistic about the tx execution itself - the `send_raw_transaction` is executed the same way as normal `send_raw_transaction`.

So I would call it something like `send_raw_transaction_with_detailed_output` or something like `send_raw_transaction_return_execution_logs`.

Reply:
Changed to `send_raw_transaction_with_detailed_output`